packages/hash/api/src/graphql/typeDefs/paginationConnections.typedef.ts

```diff
@@ -0,0 +1,58 @@
+import { gql } from "apollo-server-express";
+
+export const pagePagination = gql`
+  """
+  Each pagination specifies if there are more results to request.
+  Please use the 'nextPageCursor' for every subsequent pagination request with the 'after' parameter.
+  """
+  type PageInfo {
```

Comment:
I don't think this blocks merging, but would it be any trouble to add `pageCount` (or `numberOfPages`) as one of the `PageInfo` fields?

I appreciate that adding `hadPreviousPage` and `previousPageCursor` is not possible when using the opensearch implementation of pagination (based on internal prior discussions)

Reply:
Adding this after ironing out comments above :)